test/hotspot/jtreg/compiler/lib/template_framework/README.md

```diff
@@ -0,0 +1,8 @@
+# Template Framework
+The Template Framework allows the generation of code with Templates. The goal is that these Templates are easy to write, and allow regression tests to cover a larger scope, and to make temlate based fuzzing easy to extend.
+
+The Template Framework only generates code in the form of a String. This code can then be compiled and executed, for example with help of the [Compile Framework](../compile_framework/README.md).
+
+The basic functionalities of the Template Framework are described in the [Template Class](./Template.java), together with some examples. More examples can be found in [TestSimple.java](../../../testlibrary_tests/template_framework/examples/TestSimple.java) and [TestTutorial.java](../../../testlibrary_tests/template_framework/examples/TestTutorial.java).
+
+The [Template Library](../template_library/README.md) provides a large number of Templates which can be used to create anything from simple regression tests to complex fuzzers.
```

Comment:
`template_library/README.md` does not exist.

Another thought: Should `template_library` be a subfolder in `template_framework`? Otherwise, it could suggest to be something separate from the Template Framework.